test/wt_hang_analyzer/wt_hang_analyzer.py

```diff
@@ -626,5 +632,21 @@ def main():
     if trapped_exceptions:
         sys.exit(1)
 
+# Remove the lowest bit from the core dump mask. The lowest bit is for
+# anonymous private mappings (see: `man core`). These mappings are used by
+# ASAN, MSAN et al to allocate shadow memory. Some of these mappings are
+# correctly marked as "don't dump" via `memadvise`, but not
```

Comment:
I would suggest trying to keep the comment padding roughly the same so you could have "all" on this line, but probably not unfortunately. It doesn't matter that much in a python script though. More of a .c file concern.

Reply:
fixed in 49ee77f